app/src/main/java/org/fossasia/susi/ai/forgotPassword/IForgotPasswordInteractor.kt

```diff
@@ -0,0 +1,23 @@
+package org.fossasia.susi.ai.forgotPassword
+
+import android.content.Context
+
+/**
+ * Created by meeera on 7/7/17.
+ */
+interface IForgotPasswordInteractor {
```

Comment:
Please update the fork and resolve conflicts and push it. This file is delete now and should not exist anywhere.